ape_safe/client/__init__.py

```diff
@@ -129,7 +134,7 @@ def post_transaction(
                 b"",
             )
         )
-        post_dict: dict = {"signature": to_hex(signature), "origin": ORIGIN}
+        post_dict: dict = {"signature": signature.hex() if signature else None, "origin": ORIGIN}
```

Comment:
`.hex()` doesn't include a `0x` prefix, in case it matters

Reply:
works without this change yeah